IndicatorFields/incidentfield-port.json

```diff
@@ -0,0 +1,49 @@
+{
+    "id": "indicator_port",
+    "version": -1,
+    "modified": "2019-12-29T12:32:09.901697254Z",
+    "sortValues": null,
+    "propagationLabels": null,
+    "vcShouldIgnore": false,
+    "commitMessage": "",
+    "shouldCommit": false,
+    "name": "Port",
+    "prevName": "Port",
+    "ownerOnly": false,
+    "placeholder": "",
+    "description": "",
+    "cliName": "port",
+    "type": "tagsSelect",
+    "closeForm": false,
+    "editForm": true,
+    "required": false,
+    "script": "",
+    "fieldCalcScript": "",
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "selectValues": [],
+    "validationRegex": "",
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 2,
+    "mergeStrategy": "",
+    "hidden": false,
+    "associatedTypes": [
+        "IP",
+        "IPv6"
```

Comment:
This indicator type doesn't exist yet. The PR is dependent on it.

Reply:
and on CIDR and CIDRv6